### PR TITLE
config: Remove NSS vestiges

### PR DESCRIPTION
**Pull request by jlucovsky**
Remove NSS vestiges from CI and doc.

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket: [6712](https://redmine.openinfosecfoundation.org/issues/6712)

Describe changes:
- Remove NSS from doc and CI pipelines
- Remove duplicate libjansson package from CI

### Provide values to any of the below to override the defaults.

To use a pull request use a branch name like `pr/N` where `N` is the
pull request number.

Alternatively, `SV_BRANCH` may also be a link to an
OISF/suricata-verify pull-request.

```
SV_REPO=
SV_BRANCH=
SU_REPO=
SU_BRANCH=
LIBHTP_REPO=
LIBHTP_BRANCH=
```

### CODE REVIEW

**Review by inashivb — COMMENTED** [review]
LGTM. Why is this draft?

**Comment by jlucovsky** [comment]
> LGTM. Why is this draft?

I wanted to make sure the CI pipelines were ok. I'll convert it in a moment.

File: .github/workflows/builds.yml [hunk 1]
```diff
@@ -1529,7 +1515,6 @@ jobs:
                 libjansson-dev \
                 libevent-dev \
                 libevent-pthreads-2.1-7 \
-                libjansson-dev \
```
Comment (catenacyber) [inline .github/workflows/builds.yml:1532]: We still need libjansson :-p

File: .github/workflows/builds.yml [hunk 2]
```diff
@@ -235,7 +235,6 @@ jobs:
                 libtool \
                 lz4-devel \
                 make \
-                nss-devel \
```
Comment (catenacyber) [inline .github/workflows/builds.yml:238]: I think libnspr4-dev can be removed with nss where it exists

**Review by jasonish — CHANGES_REQUESTED** [review]
Also remove libnspr...

File: .github/workflows/builds.yml (hunk 1 above)
Comment (jlucovsky) [reply .github/workflows/builds.yml:1532]: Do we need it twice? It's on line 1515

File: .github/workflows/builds.yml (hunk 2 above)
Comment (jlucovsky) [reply .github/workflows/builds.yml:238]: Will do.

**Comment by jlucovsky** [comment]
Continued in #10310

**Pull request closed without merging**